modules/ossm-release-notes-3-1-new-features-enhancements.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/ossm-release-notes/ossm-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-3-1-new-features-enhancements_{context}"]
+= {SMProductName} version 3.1 new features and enhancements
+
+This release makes {SMProductName} 3.1 generally available, adds new features, addresses Common Vulnerabilities and Exposures (CVEs), and is supported on {ocp-product-title} 4.14 and later.
+
+For a list of supported component versions and support features, see "Service Mesh 3.0 feature support tables".
+
+//For a complete list of changes between {SMProduct} 2 and {SMProduct} 3, see "Important information to know if you are migrating from OpenShift Service Mesh 2.6".
+
+[id="support-for-kubernetes-gateway-api_{context}"]
+== Support for Kubernetes Gateway API
+
+This release introduces support for {k8s} Gateway API custom resource definitions (CRDs). You can now use these CRDs to configure {SMProduct} with the {k8s} Gateway API. This feature is available with {product-title} 4.19.
+
+[id="support-for-ossm-on-x86-dual-stack-clusters_{context}"]
+== Support for OpenShift Service Mesh on x86 dual-stack clusters
+
+This release introduces support for {SMProduct} on x86 dual-stack clusters. This feature remains a technology preview on all other platforms.
+
+[id="kubernetes-traffic-distribution-support_{context}"]
+== Kubernetes Traffic Distribution Support
+
+This release introduces support for the {k8s} traffic distribution feature, part of the {k8s} Service API, within {SMProduct}. As of {product-title} 4.19, this is a Beta feature and requires enabling the `ServiceTrafficDistribution` feature.
```

Comment:
Should we explain where a customer can enable this? Maybe something like the following but check with a developer to confirm:

```suggestion
This release introduces support for the {k8s} traffic distribution feature, part of the {k8s} Service API, within {SMProduct}. As of {product-title} 4.19, this is a Beta feature and requires enabling the `ServiceTrafficDistribution` parameter in the {istio} Custom Resources (CRs).
```